packages/telestion-client-common/src/hooks/use-desktop-notifications.ts

```diff
@@ -0,0 +1,76 @@
+import { useLogger, useNotifications } from '@wuespace/telestion-client-core';
+import { useEffect } from 'react';
+
+/**
+ * Enables and handles the usage of native desktop notifications for implementing the
+ * `@wuespace/telestion-client-core`'s abstract `useNotifications` APIs.
+ *
+ * Call this hook once in your `app.tsx` and everything else gets handled for you.
+ *
+ * @see {@link @wuespace/telestion-client-core#useNotifications}
+ *
+ * @example
+ * ```ts
+ * // [...]
+ * import { useDesktopNotifications } from '@wuespace/telestion-client-common';
+ *
+ * export function App() {
+ *     useDesktopNotifications();
+ *
+ *     useEventBusManager();
+ *     // [...]
+ *
+ *     return <TelestionClient>
+ *         // [...]
+ *     </TelestionClient>
+ * }
+ * ```
+ */
+export function useDesktopNotifications(): void {
+	const logger = useLogger('Desktop Notifications');
+
+	const [notifications, isMuted, dismiss] = useNotifications(
+		notificationStore => [
+			notificationStore.notifications,
+			notificationStore.isMuted,
+			notificationStore.dismiss
```

Comment:
Can you please extract the mapping function in the zustand hook and put it above the export?
This increases performance between rerenders.

For example:
https://github.com/wuespace/telestion-client/blob/dd468b4a46b3171fa42e39806722b0d478ed80af/packages/telestion-client-common/src/hooks/abstractions/use-dashboards.ts#L12-L16